src/main/frontend/context/i18n.cljs

```diff
@@ -17,8 +18,13 @@
     (try
       (apply translate preferred-language args)
       (catch :default e
-        (js/console.error "Translating dict" e)
-        (apply translate :en args)))))
+        (log/error :failed-translation {:arguments args
+                                        :lang preferred-language})
+        (apply translate :en args)
```

Comment:
`(apply` should come at last to provide a default value.

Reply:
Good catch. Updated